src/main/java/org/elasticsearch/script/ScriptParameterParser.java

```diff
@@ -231,11 +231,4 @@ public ScriptType scriptType() {
             return scriptType;
         }
     }
-
-    public static class ScriptParameterParseException extends ElasticsearchException {
```

Comment:
I can leave this exception here and just remove it in 2.0 when this file gets deleted